internal/dcache/rpc/client/functions.go

```diff
@@ -437,6 +541,9 @@ func PutChunkDC(ctx context.Context, targetNodeID string, req *models.PutChunkDC
 	log.Debug("rpc_client::PutChunkDC: Sending PutChunkDC request to nexthop node %s and %d daisy chain RV(s): %v",
 		targetNodeID, len(req.NextRVs), reqStr)
 
+	rvName := cm.RvIdToName(req.Request.Chunk.Address.RvID)
+	common.Assert(cm.IsValidRVName(rvName), rvName)
```

Comment:
this call could b expensive for clusters with lot of RVs,
see if we can index by rvid